activate-virtualenv.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# ==========================================================================
+# Script for activating the virtualenv (creating it first, if necessary).
+# NOTE: Run this script while in the project root directory.
+#       It will not run correctly when run from another directory.
+# ==========================================================================
+
+# Set script to exit on any errors.
+set -e
+
+# Presumes the varialble VIRTUAL_ENV is already set to the name of the venv
+
+echo 'Activating virtualenv, if not already activated...'
+
+source $(brew --prefix)/bin/virtualenvwrapper.sh
```

Comment:
This assumes that this script is being run on a Mac with virtualenvwrapper installed via Brew, which prevents its usage in Vagrantbox and (potentially) Docker-based setups. Any reason this couldn't be changed to remove the reference to `brew --prefix` and just add a documented requirement that virtualenvwrapper be installed and on the default PATH?